### ISSUE TITLE
No Update of the databank after blank installation

### ISSUE THREAD

**Issue by Schneetiger** [contao/contao#232]
After installing a fresh Contao 4.4.30 with CM 1.1.2 there is no possibility to update the databank (MariaDB); it just hangs. You can click the actualisation button as often you want, there is no reaction. The php-version is 7.1. Getting into the Backend is no problem and seems to work fine so far. At the moment I can´t tell more because the website is still under devolopment.

![screenshot_2018-12-12 contao open source cms](https://user-images.githubusercontent.com/29976466/49892891-66c4ab00-fe4a-11e8-8a2a-0e9ac8455deb.png)

**Comment by leofeyer** [comment]
Please try another browser and post the console log here if it still does not work.

**Comment by Schneetiger** [comment]
Hi Leo, thanks a lot your work, support and developing Contao.

Well, after installing the updates 4.4.31 and 4.4.32 the database still wasn´t able to get updated. And yes I tried other browsers like Opera and Chrome before I deleted the first installation. The results were all the same; so I decided to start new and I think this was the right decision.
Therefore I deleted this installation and startet right from the beginning with the actual version 4.4.32. This time it works fine, no problem with the database.
Good work, Leo
